api/graphql/schema.graphql

```diff
@@ -49,3 +53,16 @@ type Query {
   person(id: ID): Person
   community(id: ID, slug: String): Community
 }
+
+input MeInput {
+  name: String
+  email: String
+  avatarUrl: String
+  bannerUrl: String
+  bio: String
+  url: String
```

Comment:
what led to the selection of these specific attributes, from the whole 'whitelist' that is under validateAndSave, and the difference between the graphql mutation and the possible mutation available via the original controller method